easy-ui-react/src/Banner/Banner.module.scss

```diff
@@ -0,0 +1,67 @@
+@use "../styles/common" as *;
+
+.variantSuccess {
+  @include component-token(
+    "banner",
+    "color",
+    theme-token("color.text.success.inverse")
+  );
+  @include component-token(
+    "banner",
+    "background_color",
+    theme-token("color.background.success")
+  );
+}
+
+.variantPrimary,
+.variantNeutral {
+  @include component-token(
+    "banner",
+    "color",
+    theme-token("color.text.inverse")
+  );
+}
+
+.variantPrimary {
+  @include component-token(
+    "banner",
+    "background_color",
+    theme-token("color.background.primary")
+  );
+}
+
+.variantNeutral {
+  @include component-token(
+    "banner",
+    "background_color",
+    theme-token("color.background.neutral")
+  );
+}
+
+.Banner {
```

Comment:
**Question:** Are we missing setting a max width and margin? There's a note in the Figma file that says

> Notes:
Banner is centered with 48px left and right padding. Banner width adjusts to take up rest of width with a max width of 1348
Text wraps and expands height if needed

Reply:
- with regards to outer margins, my thinking was that this kind of thing would be handled by a layout concerned component. steve and i briefly discussed it on the spec ticket here https://github.com/EasyPost/easy-ui/pull/87#discussion_r1165672389 , but on second thought, it does give me pause since this component is quite specific in its use. curious how you feel about this though, im open to ideas!

- with regards to `max-width`, i updated the PR to handle it, but there might be a more elegant way to deal with it, i tried setting it on the banner div itself, but that didn't seem to work. i ended up adding an outer div with these styles. open to ideas if you feel theres a better approach though

```css
.bannerContentContainer {
  display: flex;
  max-width: 1348px;
}

```